src/Illuminate/Database/Schema/Grammars/ChangeColumn.php

```diff
@@ -119,17 +128,35 @@ protected static function getDoctrineColumn(Table $table, Fluent $fluent)
      */
     protected static function getDoctrineColumnChangeOptions(Fluent $fluent)
     {
-        $options = ['type' => static::getDoctrineColumnType($fluent['type'])];
+        $dcType = static::getDoctrineColumnType($fluent['type']);
```

Comment:
What `dc` prefix mean? Doctrine? It's better to write it explicitly then.

Reply:
I agree, it's fixed now.